plugins/hazelcast37/src/main/java/org/radargun/service/Hazelcast37ClientService.java

```diff
@@ -0,0 +1,57 @@
+package org.radargun.service;
+
+import com.hazelcast.client.HazelcastClient;
+import com.hazelcast.client.config.ClientConfig;
+import com.hazelcast.core.MemberAttributeEvent;
+import com.hazelcast.core.MembershipEvent;
+import com.hazelcast.core.MembershipListener;
+import org.radargun.Service;
+import org.radargun.config.Property;
+import org.radargun.traits.ProvidesTrait;
+
+/**
+ * Hazelcast client service
+ *
+ * @author Roman Macor &lt;rmacor@redhat.com&gt;
+ */
+
+@Service(doc = "Hazelcast client")
+public class Hazelcast37ClientService extends Hazelcast36Service {
```

Comment:
Well, they don't use the same API to configure embedded mode and client. Configurations API and XML configuration files are different. You can't use the same XML file to configure both client and embedded instance. But once you have Hazelcast Instance or Hazelcast client instance you can use the same API to perform operations on it.

Moving it to Hazelcast36Service would mean there would be properties which would only apply to embedded mode and properties which would only apply to client in the same service. (They would be differentiated by documentation) If that's not an issue I can move on with this.